app/models/case/base.rb

```diff
@@ -26,6 +26,7 @@
 #  type                 :string
 #  appeal_outcome_id    :integer
 #  dirty                :boolean          default(FALSE)
+#  user_id              :integer          not null, default(-100), foreign key
```

Comment:
Shouldn't this be creator_id?

Reply:
I thought these were the actual database column names rather than the virtual names

Reply:
yes they are. Would still make sense to call it creator_id IMHO. What if we needed an actual user_id for some reason?

Reply:
Trying not to think that far ahead!